src/openrct2/windows/map.c

```diff
@@ -1349,41 +1347,32 @@ static void window_map_place_park_entrance_tool_down(sint32 x, sint32 y)
  */
 static void window_map_set_peep_spawn_tool_down(sint32 x, sint32 y)
 {
-    rct_map_element *mapElement, *surfaceMapElement;
+    rct_map_element *mapElement;
     sint32 mapX, mapY, mapZ, direction;
 
     footpath_get_coordinates_from_pos(x, y, &mapX, &mapY, &direction, &mapElement);
-    if (mapX == 0x8000)
-        return;
-
-    surfaceMapElement = map_get_surface_element_at(mapX >> 5, mapY >> 5);
-    if (surfaceMapElement == NULL) {
-        return;
-    }
-    if (surfaceMapElement->properties.surface.ownership & 0xF0) {
-        return;
-    }
-
-    mapX = mapX + 16 + (word_981D6C[direction].x * 15);
-    mapY = mapY + 16 + (word_981D6C[direction].y * 15);
```

Comment:
@IntelOrca @Gymnasiast Could one of you guys give me an update if you guys were able to get that assert that i mentioned above? i'm not sure if you got the reply for this or not.

Also @Gymnasiast , in response to "have you tested this in multiplayer" Yes I did, I had to test it with localhost and advertising disabled, and tested it from the same computer, but it did work as behaved before switched it back to the old formula (which is what's confusing me above).